pkgs/by-name/mu/multipass/multipassd.nix

```diff
@@ -118,27 +107,27 @@ stdenv.mkDerivation {
     EOF
   '';
 
-  # We'll build the flutter application seperately using buildFlutterApplication
+  # We'll build the flutter application separately using buildFlutterApplication
   cmakeFlags = [ "-DMULTIPASS_ENABLE_FLUTTER_GUI=false" ];
 
   buildInputs = [
     gtest
+    kdePackages.qtbase
+    kdePackages.qtwayland
```

Comment:
I don't think that is intended to be used like that. Please stay to the things under qt

Reply:
The other option I see (for `qtbase`) is: `libsForQt5.qt5.qtbase` - which is a major release out of date, is that what this should switch to? :)

Reply:
Okay, figured it out. Using `qt6.qtbase` etc now.